packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/quanticBreadcrumbManager.js

```diff
@@ -192,21 +206,23 @@ export default class QuanticBreadcrumbManager extends LightningElement {
   }
 
   get facetBreadcrumbValues() {
-    const facetBreadcrumbsToDisplay = this.facetBreadcrumbs || [];
+    const facetBreadcrumbsToDisplay = this.facetBreadcrumbs.map(breadcrumb =>this.formatFacetBreadcrumbValue(breadcrumb)) || [];
```

Comment:
I think Array.map always returns an array so no need for the ||. Unless `facetBreadcrumbs` could be undefined in which case optional chaining should be added.